source/framework/vx_graph.c

```diff
@@ -945,20 +947,23 @@ vx_status ownGraphScheduleGraphWrapper(vx_graph graph)
 VX_API_ENTRY vx_status VX_API_CALL vxScheduleGraph(vx_graph graph)
 {
     vx_status status = (vx_status)VX_SUCCESS;
+    vx_uint32 graph_debug_zonemask;
+
+    graph_debug_zonemask = graph->debug_zonemask;
 
     if(ownIsValidSpecificReference(vxCastRefFromGraph(graph), (vx_enum)VX_TYPE_GRAPH) != (vx_bool)vx_false_e)
     {
         if (graph->is_streaming_enabled != 0)
         {
-            VX_PRINT(VX_ZONE_ERROR, "graph is already streaming\n");
+            VX_PRINT_LOCAL(VX_ZONE_ERROR, graph_debug_zonemask, "graph is already streaming\n");
```

Comment:
Also a good idea, working on implementation

Reply:
After exploring further, this feature currently cannot be added. In order to include a node's name in the target kernel instance, a significant change to the framework's architecture would be required via an additional object descriptor structure within node object descriptors to carry the name of the host-side node they are spawned from across the remote cores. This would result in a substantial performance impact that we cannot introduce at this time. However, the prints will still include the function and line number from which they are called to provide additional information. 
Ex:
VX_ZONE_WARNING: [**ownAddTargetKernelInternal:449**] May need to increase the value of TIVX_TARGET_KERNEL_MAX in tiovx/include/TI/tivx_config.h